src/app/components/feed/feed-connection.factory.js

```diff
@@ -64,7 +64,7 @@
         options = options || {};
 
         var media = {videoRecv: false, audioRecv: false};
-        var cfg = {video: true, audio: true};
+        var cfg = {video: true, audio: false};
```

Comment:
Even more, I probably wouldn't change this value directly here. I would add a new option to `publish`, similar to the already existing `noCamera`. Let's call it, for example, `muted`. And then I would use that argument when calling `publish` (see the other comment).